docs-ref-conceptual/whats-new-overview.md

```diff
@@ -1,18 +1,25 @@
 ---
-title: What's new in Azure Command-Line Interface (CLI) | Microsoft Docs
-description: Learn what's new in the Azure CLI
+title: Azure CLI release highlights | Microsoft Docs
+description: Learn what's new in the Azure CLI. Read about new features and upcoming changes. Check here for announcements.
 author: dbradish-microsoft
 ms.author: dbradish
-ms.date: 05/21/2024
-ms.topic: conceptual
+ms.date: 08/01/2024
+ms.topic: concept-article
 ms.service: azure-cli
-ms.tool: azure-cli 
 ms.custom: devx-track-azurecli
 keywords: Azure CLI, new articles, new references, new samples, announcements
+#customer intent: As an Azure CLI developer, I want an easy way to see product change highlights and upcoming breaking change announcements.
 ---
-# What's new in the Azure CLI
 
-This page highlights new features, articles and learning paths for the Azure CLI.
+# Azure CLI release highlights
+
+This page highlights new features and upcoming changes for the Azure CLI.
+
+## Docker container image
+
+Beginning with Azure CLI version 2.67.0, to be released on November 21, 2024, the Docker container for the Azure CLI will be based on Azure Linux, formerly known as [CBL-Mariner](https://mcr.microsoft.com/en-us/product/cbl-mariner/base/core/about). When running the Azure CLI in a Docker container, always use a tag, like `cbl-mariner2.0` and do not rely on `latest`.
```

Comment:
```suggestion
Beginning with Azure CLI version 2.67.0, to be released on November 19, 2024, the default Docker container image for Azure CLI will be based on Azure Linux. To avoid any disruptions, review and update any dependencies you may have on the default Docker container image for Azure CLI.
```